src/components/CompareResults/loader.ts

```diff
@@ -228,6 +229,7 @@ export async function loader({ request }: { request: Request }) {
     newRepos,
     frameworkId,
     frameworkName,
+    view: compareView,
```

Comment:
I realized that you also need to add the view in the return value above (with the fake data)